pkgs/applications/misc/anytype/default.nix

```diff
@@ -2,13 +2,13 @@
 
 let
   pname = "anytype";
-  version = "0.35.4";
+  version = "0.35.25-beta";
```

Comment:
I can merge. Just want to know why you are updating this to an unreleased version instead to an release. Are there important fixes not present in any release?

Reply:
Updating as this is their latest version (I'm a community mod and in contact with the team). It's stable and 0.35.4 is missing quite a few features and fixes. If nothing would speak against it, I would try to keep the master branch (unstable) up to date with their latest release (including beta channel). And for the 23.05 branch I would update it only to their non beta versions.
Let me know of any questions or if I got something wrong with the unstable/stable channel.